pymysql/converters.py

```diff
@@ -309,13 +309,8 @@ def through(x):
     return x
 
 
-#def convert_bit(b):
-#    b = "\x00" * (8 - len(b)) + b # pad w/ zeroes
-#    return struct.unpack(">Q", b)[0]
-#    
-#     the snippet above is right, but MySQLdb doesn't process bits,
-#     so we shouldn't either
```

Comment:
No.  There is no "right" or "wrong" about this.
DB API 2.0 doesn't specify about how to convert bit type.
So this is up to driver's decision.

In case of PyMySQL, I follow MySQLdb's behavior as possible.